tools/graph-links

```diff
@@ -38,8 +40,86 @@ excludedNodes = {
     '333532b9-b7c2-4478-9415-28a3056d58df': "reject transfer option.",
     '19c94543-14cb-4158-986b-1d2b55723cd8': "reject sip option."}
 
+# For each MicroServiceChainLink in the MCP Server, the TaskType determines
+# what code will be executed for that task.
+# See: https://wiki.archivematica.org/MCP/TaskTypes
+TASK_TYPES = {
+
+    #
+    # General
+    #
+
+    # description="one instance"
+    'one_instance': '36b2e239-4a57-4aa5-8ebc-7a29139baca6',
+
+    # description="for each file"
+    # 'for_each_file': 'a6b1c323-7d36-428e-846a-e7e819423577',
+
+    #
+    # User choice
+    #
+
+    # description="get user choice to proceed with"
+    # '?': '61fb3874-8ef6-49d3-8a2d-3cb66e86a30c',
+
+    # description="get replacement dic from user choice"
+    # '?': '9c84b047-9a6d-463f-9836-eafa49743b84',
+
+    # description="Get microservice generated list in stdOut"
+    # '?': 'a19bfd9f-9989-4648-9351-013a10b382ed',
+
+    # description="Get user choice from microservice generated list"
+    # '?': '01b748fe-2e9d-44e4-ae5d-113f74c9a0ba',
+
+    #
+    # Unit variables
+    #
 
-def addArrow(sourceUUID, destUUID, color="black", label=None):
+    # description="linkTaskManagerSetUnitVariable"
+    'unit_var': '6f0b612c-867f-4dfd-8e43-5b35b7f882d7',
+    # description="linkTaskManagerUnitVariableLinkPull"
+    'unit_var_pull': 'c42184a3-1a7f-4c4d-b380-15d8d97fdd11',
+
+    #
+    # Deprecated
+    #
+
+    # description="goto magic link"
+    # '?': '6fe259c2-459d-4d4b-81a4-1b9daf7ee2e9',
+
+    # description="assign magic link"
+    'assign_magic_link': '3590f73d-5eb0-44a0-91a6-5b2db6655889',
+
+}
+
+logger = logging.getLogger()
+ch = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter('%(message)s')
+ch.setFormatter(formatter)
+logger.addHandler(ch)
```

Comment:
I'm a fan of [`logging.dictConfig`](https://docs.python.org/2/library/logging.config.html#logging.config.dictConfig), but [`logging.basicConfig`](https://docs.python.org/2/library/logging.html#logging.basicConfig) will probably be able to do this easier.